versionedRouter.js

```diff
@@ -878,10 +879,11 @@ async function handleProxyRequest(destination, ctx) {
     return {};
   };
 
-  const destinationRequest = ctx.request.body;
+  const { metadata, destinationRequest } = ctx.request.body;
   const destNetworkHandler = networkHandlerFactory.getNetworkHandler(
     destination
   );
+  const authCache = v0Utils.getDestAuthCacheInstance(destination);
```

Comment:
https://github.com/rudderlabs/rudder-transformer/pull/1484#discussion_r1001360629

Reply:
This can be moved to be inside of the responseHandler itself. Can you once verify ? @ujjwal-ab